ros/config.py

```diff
@@ -13,7 +13,7 @@
 INVENTORY_HOST = os.getenv("INVENTORY_HOST", "localhost")
 INVENTORY_PORT = os.getenv("INVENTORY_PORT", "8081")
 INVENTORY_URL = f"http://{INVENTORY_HOST}:{INVENTORY_PORT}"
-INVENTORY_EGRESS_TOPIC = os.getenv(
+INVENTORY_EVENTS_TOPIC = os.getenv(
     "INVENTORY_EGRESS_TOPIC", "platform.inventory.events")
```

Comment:
I guess you missed renaming it here?
```suggestion
    "INVENTORY_EVENTS_TOPIC", "platform.inventory.events")
```